minio/api.py

```diff
@@ -795,6 +802,15 @@ def put_object(self, bucket_name, object_name, data, length,
         if length > MAX_MULTIPART_OBJECT_SIZE:
             raise InvalidArgumentError('Input content size is bigger '
                                        ' than allowed maximum of 5TiB.')
+
+        if part_size < DEFAULT_PART_SIZE:
```

Comment:
This should be 'if part_size < MIN_PART_SIZE`, we cannot forbid a 5mb part if we set 64Mb as default part size in the future.

Reply:
That is the intention @vadmeste - why would want to go lower than DEFAULT there is no good reason.

Reply:
But i can keep for sanity sake.. MIN_PART_SIZE